xExtension-showFeedID/extension.php

```diff
@@ -5,6 +5,22 @@
 final class ShowFeedIdExtension extends Minz_Extension {
 	#[\Override]
 	public function init(): void {
-		Minz_View::appendScript($this->getFileUrl('showfeedid.js'));
+		if (Minz_Request::paramString('c') === 'subscription') {
+			$this->registerTranslates();
+			$this->registerHook('js_vars', [$this, 'jsVars']);
+			Minz_View::appendScript($this->getFileUrl('showfeedid.js'));
+		}
+	}
+
+	/**
+	 * @param array<mixed> $vars
+	 * @return array<mixed>
```

Comment:
It would be nice to be a bit more precise if possible, like `array<string,string|array<string, mixed>>` or something like that